docusaurus.config.js

```diff
@@ -174,5 +164,4 @@ module.exports = {
       },
     ],
   ],
-  themes: ['@docusaurus/theme-live-codeblock'],
```

Comment:
Why was this removed

Reply:
this was added before to address an issue that code blocks in light mode would have the "Copy" button almost invisible. With this change the Copy button started showing up properly. (P.s it's not what the plugin itself is meant to do, it's actually a plugin that allows to modify the code in code blocks and changes some styling)

In newer versions of docusaurus the copy button is showing up fine out of the box, so i cleaned the code up
![image](https://user-images.githubusercontent.com/11838981/90051887-3c656980-dc8d-11ea-9300-23e3809be07c.png)